src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const properties = sourceString.split(';');
+
+  properties.forEach((element) => {
+    let [property, value] = element.split(':');
+
+    if (property && value) {
+      property = property.trim();
+      value = value.trim();
+
+      result[property] = value;
```

Comment:
There's a potential issue where CSS properties with vendor prefixes might have spaces within the property name, e.g., '- webkit-border-radius'. Ensure that the property name is properly trimmed to handle such cases.